src/app/pipes/phone.pipe.spec.ts

```diff
@@ -1,15 +1,15 @@
 import { PhonePipe } from './phone.pipe';
-import { MockBuilder, MockRender } from 'ng-mocks';
+import {MockBuilder, MockRender, ngMocks} from 'ng-mocks';
 describe('Phone Pipe', () => {
   beforeEach(() => {
     return MockBuilder(PhonePipe).provide(PhonePipe)
   })
   it('should be defined', () => {
     const fixture = MockRender(PhonePipe);
-    expect(fixture.componentInstance).toBeDefined();
+    expect(fixture.point.componentInstance).toBeDefined();
   })
   it('should test the transform value', () => {
-    const fixture = MockRender(PhonePipe);
-    expect(fixture.componentInstance.transform('4161234567')).toBe('+1(416)-123-4567');
+    const fixture = MockRender('{{ "4161234567" | phone }}');
+    expect(ngMocks.formatText(fixture)).toBe('+1(416)-123-4567');
```

Comment:
this one is a bug, I'll fix it today.

Reply:
Cool, let me know if there is a better way for this.

Reply:
found the issue, it's because the pipe was declared and provided, the last action won. Added a fix to respect both settings, so the pipe will stay declared and will be additionally provided.

Reply:
done. 13.5.1 has a fix for it, I've updated the PR.

Reply:
cool, thank you, let me take a look